### DISCUSSION TITLE
Not found options to validate Runway config file

### DISCUSSION THREAD

**Discussion by pbalajiips** [onicagroup/runway#1803] [General]
I want to validate my runway.yml has valid configuration. is there any way to do ? if someone know please share the commandline.

**Comment by ITProKyle** [comment]
There isn't a command to do this yet but it is in the backlog (#515).

Until then, the schema can be exported to a file using [`runway schema runway`](https://docs.onica.com/projects/runway/en/stable/commands.html#schema-runway) for use with your preferred method of schema validation (e.g. the [`redhat.vscode-yaml`](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) vscode extension)
